packages/core/src/singleton/Analytics/types.ts

```diff
@@ -2,5 +2,8 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { PinpointProviderConfig } from '../../providers/pinpoint/types';
+import { KinesisProviderConfig } from '../../providers/kinesis/types';
 
-export type AnalyticsConfig = PinpointProviderConfig;
+export type AnalyticsConfig = Partial<
+	PinpointProviderConfig & KinesisProviderConfig
+>;
```

Comment:
Nit: Would prefer to just make `Kinesis` & `Pinpoint` optional in their respective config types.

```suggestion
export type AnalyticsConfig = PinpointProviderConfig & KinesisProviderConfig;
```

Reply:
will update it.